modules/objdetect/src/barcode.cpp

```diff
@@ -142,11 +142,15 @@ struct BarcodeImpl : public GraphicalCodeDetector::Impl
 public:
     shared_ptr<SuperScale> sr;
     bool use_nn_sr = false;
+    double detectorThrDownSample = 512.f;
+    vector<double> detectorWindowSizes = {0.01f, 0.03f, 0.06f, 0.08f};
```

Comment:
Should be vector of float rather than double

Reply:
Done.